tests/cases/fourslash/refactorConvertArrowFunctionOrFunctionExpression_ToNamed_SingleLine.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path='fourslash.ts' />
+
+//// let foo = /*x*/(/*y*/) => 1 + 1;

```

Comment:
There should be tests where the function expression isn't preceded by a variable declaration.